releasing/centos.dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG _VERSION_
+
+FROM centos:$_VERSION_ AS base
+
+# Install basic tools
+RUN yum install -y \
+    file           \
+    git            \
+    redhat-lsb     \
+    tar            \
+    wget
+
+#--
+
+FROM base AS eight
+
+# Install C++ compiler
+RUN yum -y group install --nogpgcheck "Development Tools" || \
+    yum -y groupinstall --nogpgcheck "Development Tools"
+
+RUN yum install -y bison flex
+
+RUN gcc --version
+RUN g++ --version
+
+#--
+
+FROM base
+
+# Link libstdc++ statically so people don't have to install devtoolset-8
+# just to use verible.
+ENV BAZEL_LINKOPTS "-static-libstdc++:-lm -static-libstdc++:-lrt"
+ENV BAZEL_LINKLIBS "-l%:libstdc++.a"
+
+# Get a newer GCC version
+RUN yum install -y --nogpgcheck centos-release-scl
+RUN yum install -y --nogpgcheck devtoolset-8
+
+RUN yum install -y bison flex
+
+# Build a newer version of flex
```

Comment:
This is only needed for some versions of CentOS right?

Reply:
Yes. That dockerfile has theree stages:

- The first stage (L1-L12) is used regardless of the OS version.
- The second stage (L14-L25) is executed on CentOS 8 only.
  - Others use the third stage right below (L27-L53).

Reply:
The dockerfile was renamed to `centos.basic.dockerfile`.